client/styles/components/_help-modal.scss

```diff
@@ -0,0 +1,39 @@
+.help-modal {
+  @extend %modal;
+  padding: #{20 / $base-font-size}rem;
+  width: #{500 / $base-font-size}rem;
+}
+
+.help-modal__header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.help-modal__section {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: #{10 / $base-font-size}rem 0;
+
+  // Basic text styles for help body text
+  ul,
```

Comment:
oops my bad, did not see that they were nested in `.help-modal__section`. this is for a different PR, but i think a lot of the overlay/modal styles could probably be cleaned up and made more generic 😄